### DISCUSSION TITLE
Report Issue 🪲

### DISCUSSION THREAD

**Discussion by DmitryPopov** [wanderer-industries/wanderer#3] [Q&A]
If you found some strange behaviour or some functionality that should be improved, feel free to answer on this topic, we'll take care on it.

**Comment by patto-t** [comment]
I have used your installation instructions on three different Linux distros -- Amazon Linux, Ubuntu, and CentOS, and had problems on all 3. I can install not only the hello world Docker container and have it execute successfully, but I can also install other Docker-based images and start them just fine.

I cloned the GitHub repo, created the dev app in the Eve Dev section, and edited the env file to replace the required values including creating my own keys:
![image](https://github.com/user-attachments/assets/099623a8-3241-407a-9b94-a1473787ed89)

When I run the "docker compose -f docker-compose.yml up -d" command, it seems to build fine, except for a message at the end about network "web" declared as external, but could not be found:
![image](https://github.com/user-attachments/assets/ecf35b55-f3eb-4556-9fd9-96eecfb155b5)

There are no containers running, Docker is doing nothing, no images were built, no files exist:
![image](https://github.com/user-attachments/assets/05e57e3a-a07b-4c93-92cf-a467c55e75fe)

Here's me trying to down and re-up it:
![image](https://github.com/user-attachments/assets/4b282de9-dd77-41d5-8900-388ad112e9db)

But if I do a "docker compose down --rmi all --volumes" this happens:
![image](https://github.com/user-attachments/assets/d13fd546-dfab-4d43-98d2-c2ef9d64ce60)

There are no log entries for Docker because no containers are starting or attempting to start.

Please help? Thanks.

**Reply by patto-t** [reply]
Oh and needless to say, I have tried to connect with Lynx to http://localhost:8000 and there's nothing to connect to.
![image](https://github.com/user-attachments/assets/ed4b244c-5500-4656-a25d-e328caf1929b)

**Comment by DmitryPopov** [comment]
> about network "web" declared as external, but could not be found

Try to add network **web** before start docker-compose by running:
`docker network create "web"`

**Reply by patto-t** [reply]
That worked! Thanks!

**Reply by patto-t** [reply]
I came back here again for this :)
